inspirehep/modules/records/mappings/records/journals.json

```diff
@@ -204,6 +204,10 @@
                     ],
                     "tokenizer": "standard",
                     "type": "custom"
+                },
+                "lowercase_analyzer": {
+                    "filter": "lowercase",
+                    "tokenizer": "keyword"
```

Comment:
We use them in the UI in the canonical form, like `Authors`. But then you can do `/search?cc=Authors` or `/search?cc=authors`